src/lib.rs

```diff
@@ -366,6 +368,96 @@ pub trait PrimeFieldRepr: Sized +
     /// Performs a leftwise bitshift of this number, effectively multiplying
     /// it by 2. Overflow is ignored.
     fn mul2(&mut self);
+
+    /// Writes this `PrimeFieldRepr` as a big endian integer. Always writes
+    /// `(num_bits` / 8) bytes.
```

Comment:
``num_bits`` is documented as "Compute the number of bits needed to encode this number."; if that's guaranteed to be a multiple of 8 then it doesn't say so.